### PR TITLE
Feature right click menu alignment

### PR DESCRIPTION
**Pull request by rkausch**
Fixes #5964

## Proposed Changes
  - Moved menu declaration from enum into extensible command framework

### CODE REVIEW

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    /**
+     * Default set of commands.
+     */
+    private static final Map<String, SelectionCommand> DEFAULT_COMMAND_MAP = DEFAULT_COMMANDS.stream()
+            .collect(Collectors.toMap(c -> c.getName(), c -> c));
+//
+//    /**
+//     * Default set of commands.
+//     */
+//    private static final Map<String, SelectionCommand> CLASSNAME_COMMAND_MAP = DEFAULT_COMMANDS.stream()
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: Do you want to leave these commented blocks as is?

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    /**
+     * Gets the point menu items.
+     *
+     * @param al the {@link ActionListener}
+     * @param hasFilters Whether there are filters associated with the menu.
+     * @return the point menu items
+     */
+    public static List<Component> getPointMenuItems(ActionListener al, boolean hasFilters)
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: Unnecessary argument hasFilters?

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+    }
+
+    /**
+     * Gets the polyline menu items.
+     *
+     * @param al the {@link ActionListener}
+     * @param hasFilters Whether there are filters associated with the menu.
+     * @return the polyline menu items
+     */
+    public static List<Component> getPolylineMenuItems(ActionListener al, boolean hasFilters)
```
Comment (Xenotide) [inline open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: This function also doesn't use hasFilters.

Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: fixed

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java (hunk 2 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: fixed

File: open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java (hunk 1 above)
Comment (rkausch) [reply open-sphere-base/mantle/src/main/java/io/opensphere/mantle/plugin/selection/SelectionCommandFactory.java]: fixed

**Pull request merged**